src/app/components/hmcts-global-header/hmcts-global-header.component.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { HttpClientModule } from '@angular/common/http';
```

Comment:
@johnbenjamin-hmcts - this should be using `HttpClientTestingModule` for testing. I also made the same mistake and @paddybasi advised to use the above. I have committed the fix to the `feature/global-search` branch; please pull the latest.

Reply:
Thank you @Daniel-Lam - I have now pulled the latest from the base branch and updated the code.